content/en/apps/guides/forms/rdtoolkit-integration.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Rapid Diagnostic Toolkit Integration"
+linkTitle: "RDToolkit Integration"
+weight:
+description: >
+  Integration for the Rapid Diagnostic Toolkit (RDToolkit)
+keywords: RDToolkit RDT
+
+---
+
+_Introduced in v3.12.0_
+
+CHT has an integration with [Rapid Diagnostic Toolkit (RDToolkit)](https://github.com/dimagi/rd-toolkit) which is an Android app developed by Dimagi that facilitates the use of Rapid Diagnostic Test (RDT) in global health care. 
+The integration is only available in the CHT Android app and consists of two Enketo widgets (rdtoolkit-provision and rdtoolkit-capture) that are used in XForms to provision RDT and capture results.
+
+## Create the XForms
+It is required to create one form for provisioning RDT and one form for capturing RDT results, see [the documentation](https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/) for more details about app forms.
+
```

Comment:
Out of curiosity, what happens if you have the provision and capture fields in the same form?

Reply:
Good question, if we put both widgets next to each other (to simulate pages), the pyxform return errors because the groups and fields have similar name. I guess if we merge the 2 into one group then probably we'll have 2 buttons displaying at the same time (provision and capture), where after provisioning will have to wait until the capture is ready to be able to safe the form..  but keeping both in the same form will require a bit of work 🤔